homeassistant/components/comelit/sensor.py

```diff
@@ -80,3 +150,34 @@ def native_value(self) -> StateType:
             self.coordinator.data[OTHER][self._device.index],
             self.entity_description.key,
         )
+
+
+class ComelitVedoSensorEntity(CoordinatorEntity[ComelitVedoSystem], SensorEntity):
+    """Sensor device."""
+
+    _attr_has_entity_name = True
+    _attr_name = None
+    entity_description: ComelitVedoEntityDescription
+
+    def __init__(
+        self,
+        coordinator: ComelitVedoSystem,
+        zone: ComelitVedoZoneObject,
+        config_entry_entry_id: str,
+        description: ComelitVedoEntityDescription,
+    ) -> None:
+        """Init sensor entity."""
+        self._api = coordinator.api
+        self._zone = zone
+        super().__init__(coordinator)
+        # Use config_entry.entry_id as base for unique_id
+        # because no serial number or mac is available
+        self._attr_unique_id = f"{config_entry_entry_id}-{zone.index}"
+        self._attr_device_info = coordinator.platform_device_info(zone, "zone")
+
+        self.entity_description = description
+
+    @property
+    def native_value(self) -> StateType:
+        """Sensor value."""
+        return self.entity_description.value(self.coordinator, self._zone.index)
```

Comment:
Are you planning on adding more Vedo sensors? Because now you are using entity descriptions but all Vedo sensors are using the same logic, so you could move that logic from the entity description to this function

Reply:
Done, no plan for now. Was just a future preparation